sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -3211,6 +3217,8 @@ export const StorageOAuthScopes: string | string[];
 
 // @public
 export interface StoragePipelineOptions {
+    // (undocumented)
+    audience?: string | string[];
```

Comment:
Let's remove `string[]` as @xirzec suggested.

Reply:
Oh, Jeff suggested keeping `string[]` instead.  However, I cannot think of a scenario where we would specify more than one?

Reply:
I am not familiar enough with the scenario. I have no strong feelings, just picking one or the other feels simpler.